database/migrations/model/Version20240513155541.php

```diff
@@ -0,0 +1,52 @@
+<?php namespace Database\Migrations\Model;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Table;
+use LaravelDoctrine\Migrations\Schema\Builder;
+/**
+ * Class Version20240513155541
+ * @package Database\Migrations\Model
+ */
+class Version20240513155541 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function up(Schema $schema):void
+    {
+        $builder = new Builder($schema);
+        if($builder->hasTable("SummitScheduleConfig") && $builder->hasColumns("SummitScheduleConfig", ["TimeFormat"])) {
+            $sql = <<<SQL
+ALTER TABLE SummitScheduleConfig MODIFY TimeFormat enum('12h', '24h') DEFAULT '12h';
+SQL;
+            $this->addSql($sql);
+
+            $sql = <<<SQL
+       update SummitScheduleConfig set TimeFormat = '12h' where TimeFormat IS NULL;
```

Comment:
this is not necessary please review comment https://github.com/OpenStackweb/summit-api/pull/257/files#r1598874496